### PR TITLE
register all features without exception for those that start with 'n_'

### PR DESCRIPTION
**Pull request by asanin-epfl**

### CODE REVIEW

File: neurom/features/tests/test_get_features.py [hunk 1]
```diff
@@ -490,7 +490,10 @@ def test_segment_meander_angles_single_section():
 def test_neurite_features_accept_single_tree():
     for f in NEURITEFEATURES:
         ret = get_feature(f, NRN.neurites[0])
-        nt.ok_(ret.dtype.kind in ('i', 'f'))
+        if isinstance(ret, np.ndarray):
+            nt.ok_(ret.dtype.kind in ('i', 'f'))
+        else:
+            nt.assert_true(np.isscalar(ret))
         if len(features._find_feature_func(f).shape) >= 1:
             nt.ok_(len(ret) > 0)
```
Comment (adrien-berchet) [inline neurom/features/tests/test_get_features.py:498]: I propose the following simplification:
```suggestion
        if isinstance(ret, np.ndarray):
            nt.ok_(ret.dtype.kind in ('i', 'f'))
            nt.ok_(len(features._find_feature_func(f).shape) >= 1)
            nt.ok_(len(ret) > 0)
        else:
            nt.assert_true(np.isscalar(ret))
```

Also, shouldn't we either use `nt.assert_true`, either always `nt.ok_`?

Comment (asanin-epfl) [reply neurom/features/tests/test_get_features.py:498]: Very good. Lets use it. I've adjusted your suggestion to replace `assert_true` with `ok_`

**Review by adrien-berchet — APPROVED** [review]
Looks good!

**Comment by asanin-epfl** [comment]
very well then, thank you

**Comment by asanin-epfl** [comment]
@tomdele the PR is reviewed already. Because Adrien does not have permission to approve, can you please do it instead?

**Review by tomdele — APPROVED** [review]
Legit !

**Comment by asanin-epfl** [comment]
Thank you!

**Pull request merged**